setup.py

```diff
@@ -39,11 +39,12 @@ def get_version(module, path):
 )
 
 INSTALL_REQUIRES = [
-    'sphinx==2.0.*',
+    'sphinx>=2.0.*,<3.0.0',
```

Comment:
Just wondering.... is there a reason why it wouldn't work with a strict version of sphinx>2.0.*, is there something in sphinx>=3.0.0 that would stop it working?

Reply:
Sooooo,

These docs do not have any especially strict coupling to Sphinx, we aren't using bleeding edge features or anything like that.

However, the dependent cylc-sphinx-extensions, by it's nature as a Sphinx extension has much tighter coupling. So by making the sphinx requirement vague in cylc-doc we are effectively allowing cylc-sphinx-extensions to pick the version, which, unfortunately, is necessary due to a downstream dependency being broken at Sphinx 2.4 😞.